src/DistributedLock.Redis/RedisDistributedSemaphore.cs

```diff
@@ -39,6 +39,19 @@ public RedisDistributedSemaphore(RedisKey key, int maxCount, IDatabase database,
     /// </summary>
     public string Name => this.Key.ToString();
 
+    public int GetCurrentCount()
```

Comment:
These are public methods; please add doc comments to each. In this case I think we can just say that this functionality is equivalent to `SemaphoreSlim.CurrentCount`